src/main/java/com/sourcery/gymapp/backend/workout/model/RoutineLike.java

```diff
@@ -0,0 +1,31 @@
+package com.sourcery.gymapp.backend.workout.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import org.springframework.data.annotation.CreatedDate;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "routine_like", schema = "workout_data", uniqueConstraints = {
+        @UniqueConstraint(columnNames = {"routine_id", "user_id"})
+})
+public class RoutineLike {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
```

Comment:
Why are you creating additional id, if you are already using unique constraint. I think you are not using it anywhere